src/types/RegistrationDataType.ts

```diff
@@ -0,0 +1,35 @@
+export type GetRegistrationDataResponse = {
+  id: string;
+  createdAt: Date;
+  updatedAt: Date;
+  registrationId: string;
+  registrationFieldId: string;
+  value: string;
+}[];
+
+export type PostRegistrationDataRequest = {
+  status: 'DRAFT' | 'APPROVED' | 'PUBLISHED' | null;
```

Comment:
status type can be found as `'DRAFT' | 'APPROVED' | 'PUBLISHED' | null;` or `'DRAFT' | 'APPROVED' | 'PUBLISHED' | undefined;`

Will it be good to have some enum type for status only?

Reply:
sounds like a good idea. added that